include/grpcpp/impl/codegen/config_protobuf.h

```diff
@@ -83,6 +88,17 @@ typedef GRPC_CUSTOM_SERVICEDESCRIPTOR ServiceDescriptor;
 typedef GRPC_CUSTOM_SIMPLEDESCRIPTORDATABASE SimpleDescriptorDatabase;
 typedef GRPC_CUSTOM_SOURCELOCATION SourceLocation;
 
+namespace util {
+typedef GRPC_CUSTOM_UTIL_STATUS Status;
+
+inline util::Status JsonStringToMessage(const std::string& input,
```

Comment:
Should this use `grpc::string`?

Reply:
I think `std::string` is the right choice here. `JsonStringToMessage` takes one in, and since we are in C++ land here, it is always safe to use